scm/ipa-x-sampa.scm

```diff
@@ -45,6 +45,7 @@
 ((("1" )) ("ɨ"))
 ((("2" )) ("ø"))
 ((("3" )) ("ɜ"))
+((("3" "`" )) ("ɝ"))
```

Comment:
I confirmed that your changes are correct except this based on https://en.wikipedia.org/wiki/X-SAMPA .
It seems that ```3` ``` doesn't exist. (```E` ``` doesn't exist too.)
Could you share a definition of ```3` ```?

Reply:
Wikipedia doesn't list `` 3` `` as a separate entry, but it has an example of ``[n3`s]`` for GenAm pronunciation of _nurse_ which would be spelled [nɝs](https://en.wiktionary.org/wiki/nurse#:~:text=key%29%3A-,/n%C9%9Ds/,-Audio%20) in Unicode IPA. In the original paper introducing the notation (linked on Wikipedia), Wells gave the example of GenAm _bird_ in Kenyon and Knott transcription (ie. [bɝd](https://archive.org/details/pronouncingdicti00unse/page/48/mode/2up?view=theater&q=bird)) as ``/b3`d/``.

Wells defined `` ` `` as a diacritic and all entries that have `` ` `` in them can be analysed as a code for some base symbol followed by `` ` ``. In case of vowels, this directly corresponds to the rhotic diacritic of IPA. See https://en.wikipedia.org/wiki/R-colored_vowel#Notation. Compare how ɚ `` @` `` doesn't even appear on the current IPA chart as a standalone symbol, but only as an example of the usage of the rhoticity diacritic.
So even if it's not listed explicitly, `` E` `` unambiguously stands for ɛ˞ while ɝ is `` 3` `` (as seen in examples). The former can be entered as ɛ `E` followed by ◌˞ `` ` ``, but `` 3` `` is virtually always written with a single character ɝ rather than as a sequence ɜ˞ (ie. ɜ ``3`` followed by ◌˞ `` ` ``). Although Unicode doesn't define any equivalence, from the IPA point of view ɝ is functionally just like a precomposed character for ɜ˞.

Reply:
Thanks.

I'll merge this.